app/lib/onlineTasksMapper.js

```diff
@@ -0,0 +1,17 @@
+function map(gpData) {
+  let onlineTasks;
+
+  if (gpData.bookingSystem) {
+    onlineTasks = {};
+    onlineTasks.bookOnlineLink = gpData.bookingSystem.bookOnlineLink;
+  }
+
+  if (gpData.onlineServices && gpData.onlineServices.repeatPrescriptions) {
+    onlineTasks = onlineTasks || {};
+    onlineTasks.repeatPrescriptionOnlineLink = gpData.onlineServices.repeatPrescriptions.url;
+  }
+
+  return onlineTasks;
+}
+
+module.exports = map;
```

Comment:
I'm going to move the booking online member into online services soon. Probably at the same time as adding the online records. I'll then remove the `bookingSystem` member